src/electron/views/device-connect-view/components/android-setup/prompt-install-service-step.tsx

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import {
+    DeviceDescription,
+    DeviceDescriptionProps,
+} from 'electron/views/device-connect-view/components/android-setup/device-description';
+import { PrimaryButton } from 'office-ui-fabric-react';
+import * as React from 'react';
+import { AndroidSetupStepLayout, AndroidSetupStepLayoutProps } from './android-setup-step-layout';
+import { CommonAndroidSetupStepProps } from './android-setup-types';
+
+export const PromptInstallServiceStep = NamedFC<CommonAndroidSetupStepProps>(
+    'PromptInstallServiceStep',
+    (props: CommonAndroidSetupStepProps) => {
+        const onCancelButton = () => {
+            // To be implemented in future feature work
+            console.log(`androidSetupActionCreator.cancel()`);
+        };
+
+        const onInstallButton = () => {
+            // To be implemented in future feature work
+            console.log(`androidSetupActionCreator.installService()`);
+        };
+
+        const layoutProps: AndroidSetupStepLayoutProps = {
+            headerText: 'Let us install the latest version on your device',
+            leftFooterButtonProps: {
+                text: 'Cancel',
+                onClick: onCancelButton,
+            },
+            rightFooterButtonProps: {
+                text: 'Next',
+                disabled: true,
+                onClick: null,
+            },
+        };
+
+        const descriptionProps: DeviceDescriptionProps = props.androidSetupStoreData.selectedDevice;
+
+        return (
+            <AndroidSetupStepLayout {...layoutProps}>
+                <React.Fragment>
```

Comment:
Same as earlier comment, the `<React.Fragment>` wrapper isn't really doing anything here; I'd recommend either using a `<p>` tag instead or just omitting it

Reply:
Still had to keep <> and </> around the text or it wouldn't compile, but removed the React.Fragment